.jenkins/infrastructure/docker/build_mystikos_docker_images.Jenkinsfile

```diff
@@ -0,0 +1,139 @@
+def containers_reqs = [
+    'libcxx': ['dir': 'tests/libcxx', 'make': 'libcxx-tests'],
+    'libcxx2': ['dir': 'tests/libcxx2', 'make': 'libcxx-tests']
+]
+
+def containers_to_build = [
+    'aspnetcore5': ['dir': 'tests/aspnetcore5', 'file': 'Dockerfile.aspnet5'],
+    'aspnetcore6': ['dir': 'tests/aspnetcore6', 'file': 'Dockerfile'],
+    'dotnet-sos-prereqs': ['dir': 'tests/dotnet-sos', 'file': 'Dockerfile-prereqs'],
+    'glibc': ['dir': 'tests/glibc', 'file': 'Dockerfile'],
+    'libcxx': ['dir': 'tests/libcxx', 'file': 'Dockerfile'],
+    'libcxx2': ['dir': 'tests/libcxx2', 'file': 'Dockerfile'],
+    'pytorch-test': ['dir': 'solutions/pytorch_tests', 'file': 'Dockerfile'],
+    'dotnet_lib_glibc':   ['dir': 'solutions/dotnet_lib', 'file': 'Dockerfile.glibc'],
+    'dotnet_lib_musl':   ['dir': 'solutions/dotnet_lib', 'file': 'Dockerfile.musl']
+]
+
+pipeline {
+    agent {
+        label 'ACC-1804-DC4'
+    }
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+    }
+    parameters {
+        string(name: "MYST_VERSION", description: "Mystikos release version (Example: 0.5.0). See https://github.com/deislabs/mystikos/releases for release versions")
+        string(name: "REPOSITORY_NAME", defaultValue: "deislabs/mystikos", description: "GitHub repository to checkout")
+        string(name: "BRANCH_NAME", defaultValue: "master", description: "The branch used to checkout the repository")
+        string(name: "MYST_BASE_CONTAINER_TAG", defaultValue: "latest", description: "The tag for the new Mystikos base Docker container.")
+        string(name: "OE_BASE_CONTAINER_TAG", defaultValue: "SGX-2.15.100", description: "The tag for the base OE Docker container. Use SGX-<version> for releases. Example: SGX-2.15.100")
+        string(name: "INTERNAL_REPO", defaultValue: "https://mystikos.azurecr.io", description: "Url for internal Docker repository")
+        booleanParam(name: "PUBLISH_CONTAINERS", defaultValue: false, description: "Publish container to registry?")
+        booleanParam(name: "TAG_LATEST", defaultValue: false, description: "Update the latest tag to the currently built DOCKER_TAG")
+    }
+    environment {
+        INTERNAL_REPO_CREDS = 'mystikos-internal-container-registry'
+        // Docker plugin cannot seem to use credentials from Azure Key Vault
+        DOCKERHUB_REPO_CREDS = ''
+        BASE_DOCKERFILE_DIR = ".jenkins/docker/base/"
+    }
+    stages {
+        stage("Checkout") {
+            steps {
+                cleanWs()
+                checkout([$class: 'GitSCM',
+                    branches: [[name: BRANCH_NAME]],
+                    extensions: [],
+                    userRemoteConfigs: [[url: "https://github.com/${params.REPOSITORY_NAME}"]]])
+            }
+        }
+        stage("Base Image") {
```

Comment:
Do any of the test containers depend on the base container?

Reply:
Not that I'm aware of

Reply:
Yeah, that's a good idea 👍